pod/meeting/views.py

```diff
@@ -365,7 +371,11 @@ def render_show_page(request, meeting, show_page, direct_access):
 def join_as_moderator(request, meeting):
     """Join as a moderator."""
     try:
-        created = meeting.create(request)
+        created = True
+        if meeting.get_is_meeting_running() is not True:
+            created = meeting.create(request)
```

Comment:
"is False" serait mieux que "is not True" non ?

Reply:
non car il peut prendre d'autres valeurs ou ne rien retourner donc on execute que s'il est égal à True